test/password.cpp

```diff
@@ -0,0 +1,22 @@
+#include "catch.hpp"
+
+#include <password.h>
+
+using namespace newsbeuter;
+
+TEST_CASE("read_password()", "[password]") {
+    REQUIRE(read_password("data/empty.xml") == "");
```

Comment:
I'm not sure re-using data files for different purposes is a good idea. I suggest using `/dev/null` here, since it's guaranteed to exist and be empty on any platform Newsbeuter supports (i.e. Unix-like OSes.)